app/views/feedbacks/new.html.erb

```diff
@@ -19,7 +19,7 @@
 
 <p><%= ts("We're still in our beta phase of development, so there are ") %><a href="http://code.google.com/p/otwarchive/issues/list"><%= ts("lots of bugs to be fixed") %></a><%= ts(" and ") %><a href="https://trello.com/board/feature-requests/4f0cd935f97f91cf29246288"><%= ts("new features to be considered") %></a><%= ts(". For live updates on Archive performance, please check out our Twitter feed: ") %><a href="http://twitter.com/ao3_status">@AO3_status</a><%= ts(". Please use this form to let us know about your experience while using the site - good and bad! This form is for questions about the Archive and the OTW. Questions for authors should be left on their works. If you have technical issues with this page, please use ") %><a href="http://transformativeworks.org/contact/archive%20support"><%= ts("the back-up form.") %></a></p>
 
-<p><%= ts("<strong>We can answer Support inquiries in العربية, Bahasa Indonesia, català, čeština, Deutsch, español, français, 한국어, italiano, Kiswahili, magyar, Nederlands, polski, português, Русский, suomi, svenska, Türkçe, 中文.</strong> You can just write your question or feedback in one of the listed languages and we'll sort it out! Please allow for up to one week delay for responses in any language other than English and español.").html_safe %></p>
+<p><strong><%=ts("We can answer Support inquiries in")+' '+ArchiveConfig.SUPPORT_LANGUAGES %></strong>. <%=ts("You can just write your question or feedback in one of the listed languages and we'll sort it out! Please allow for up to one week delay for responses in any language other than English and español.").html_safe %></p>
```

Comment:
We usually use spaces around out + signs for readability: `") + ' ' + ArchiveConfig.SUPPORT_LANGUAGES`